websites/E/EternalNetwork TM/presence.ts

```diff
@@ -65,7 +65,7 @@ presence.on("UpdateData", async () => {
 			if (pathname.startsWith("/")) presenceData.state = ttl;
 
 			if (pathname.includes("/wp-admin")) {
-				presenceData.state = "Using administrating power over the website !";
+				presenceData.state = "Administrating the website";
```

Comment:
Doesn't make sense

Reply:
Changed in commit 6310c13

![Proof](https://i.imgur.com/v00LWXN.png)

Reply:
I'm not entirely sure you know how a review works. I was saying the new proposed change doesn't make sense - not the former version.